src/scripts/client.js

```diff
@@ -45,7 +45,11 @@ const main = async () => {
     // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
     // Please read the documentation at:
     // https://docs.dolby.io/communications-apis/docs/initializing-javascript
-    VoxeetSDK.initialize('customerKey', 'customerSecret');
+    VoxeetSDK.initializeToken(access_token, () => {
+      // This callback is called when the token needs to be refreshed
+    }).catch(error => {
```

Comment:
This is not an async function, you can't catch anything.

Reply:
Sounds good, will update this. We need to update this in the docs as well as it references a .catch(error)